modules/mgidBidAdapter.js

```diff
@@ -0,0 +1,232 @@
+import {registerBidder} from 'src/adapters/bidderFactory';
+import * as utils from '../src/utils';
+import * as urlUtils from '../src/url';
+import { BANNER } from 'src/mediaTypes';
+const BIDDER_CODE = 'mgid';
+const ENDPOINT_URL = '//dsp.mgid.com/prebid/';
+const VERSION = '1.0';
+
+export const spec = {
+  code: BIDDER_CODE,
+  aliases: ['mgid'], // short code
+  supportedMediaTypes: [BANNER],
+  reId: /^[0-9]+$/,
+  /**
+   * Determines whether or not the given bid request is valid.
+   *
+   * @param {BidRequest} bid The bid params to validate.
+   * @return boolean True if this is a valid bid, and false otherwise.
+   */
+  isBidRequestValid: (bid) => {
+    const banner = utils.deepAccess(bid, 'mediaTypes.banner');
+    const sizes = utils.deepAccess(banner, 'sizes');
+    let sizesOk = typeof (sizes) == 'object' && sizes.length > 0;
```

Comment:
Per LGTM result, this will always eval to true

Reply:
I cant't find no reason nor allert of LGTM for 'always evaluating to true'.
Please assist